### PR TITLE
Enhancement/duplicated entities

### PR DESCRIPTION
**Pull request by barbararochazup**
## Issue Description

Validator when create workpace or circle with a name already exists

### CODE REVIEW

File: moove/web/src/main/resources/locale/messages.properties [hunk 1]
```diff
@@ -44,3 +44,5 @@ villager.unexpected.error=An unexpected error occurred.
 idm.unexpected.error=An unexpected error occurred when trying to reach your identity manager.
 duplicated.component.name.error=You can not save two components with the same name.
 deployment.configuration.already.registered=Deployment configuration can not be registered, because it is already configuration on workspace.
+duplicated.workspace.name.error=You can not save two workspace with the same name.
+duplicated.circle.name.error=You can not save two circle with the same name.
```
Comment (fandrade-zup) [inline moove/web/src/main/resources/locale/messages.properties:48]: You need to use plural

duplicated.workspace.name.error=You can not save two **workspaces** with the same name.
￼duplicated.circle.name.error=You can not save two **circles** with the same name.

**Review by fandrade-zup — CHANGES_REQUESTED** [review]
Looks good to me, but need to use plural in messages

File: moove/web/src/main/resources/locale/messages.properties (hunk 1 above)
Comment (barbararochazup) [reply moove/web/src/main/resources/locale/messages.properties]: Done

File: moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt [hunk 2]
```diff
@@ -59,7 +59,7 @@ class MooveExceptionHandler(private val messageSource: MessageSource) {
     @ResponseBody
     fun exceptions(ex: Exception): ErrorMessageResponse {
         this.logger.error(ex.message, ex)
-        return ErrorMessageResponse.of(MooveErrorCode.INTERNAL_SERVER_ERROR, ex.message!!)
+        return ErrorMessageResponse.of(MooveErrorCode.INTERNAL_SERVER_ERROR, ex.message?.also { ex.message } ?: "Internal Server Error")
```
Comment (thallesfreitaszup) [inline moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt:62]: `ex.message ?: "Internal Server Error" `  will not produce the same output?

**Review by thallesfreitaszup — APPROVED** [review]

File: moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt (hunk 2 above)
Comment (barbararochazup) [reply moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt:62]: Sometimes message is null, this was causing a null pointer

**Review by fandrade-zup — APPROVED** [review]
LGTM

**Pull request merged**